### PR TITLE
PLAT-276: counters of pending execution

### PR DESCRIPTION
**Pull request by MikhailZhukov**
**- What I did**
Add counters of pending execution to object state

**- How I did it**
Add fields to struct `Info`

**- How to verify it**

**- Description for the changelog**
This counters must be filled when processing VStateReport (PLAT-210)

### CODE REVIEW

File: ledger-core/v2/virtual/object/object.go [hunk 1]
```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/descriptor"
 )
 
+// Stores information about Executor (key) and its pending execution requests (value)
```
Comment (ruz) [inline ledger-core/v2/virtual/object/object.go:17]: I don't think it's proper way to organise this table. It would be hard to do de-duplication checks. I, also, think that this is out of scope for this task.

Comment (MikhailZhukov) [reply ledger-core/v2/virtual/object/object.go]: Remove this queue from this PR
